pages/_app.js

```diff
@@ -166,6 +178,10 @@ export default function App({ Component, pageProps }) {
     setMarkerSize(event.target.value);
   }
 
+  function handleRange(event) {
+    setRange({ ...range, [event.target.name]: event.target.value });
+  }
+
   return (
     <>
```

Comment:
Again, no need for the React Fragment if we only return one element (i.e. Layout)

Reply:
Done.